agents/node/vcxagent-core/src/agent.js

```diff
@@ -32,12 +34,16 @@ async function createVcxAgent ({ agentName, genesisPath, agencyUrl, seed, usePos
   async function agentInitVcx () {
     logger.info(`Initializing ${agentName} vcx session.`)
     logger.silly(`Using following agent provision to initialize VCX settings ${JSON.stringify(agentProvision, null, 2)}`)
-    await initVcxCore(JSON.stringify(agentProvision))
+    logger.silly('Initializing threadpool')
+    await initThreadpool({})
+    logger.silly('Initializing issuer config')
+    await initIssuerConfig(agentProvision.issuerConfig)
```

Comment:
Can you write a small migration function for NodeJS? Should take old agent provision config and turn it into new one. Should be idempotent, so passing in config in new format should do no change.